README.md

```diff
@@ -1,47 +1,80 @@
+- Velocity and Acceleration - [carla_msgs/CarlaEgoVehicleStatus](https://github.com/carla-simulator/ros-carla-msgs/blob/master/msg/CarlaEgoVehicleStatus.msg)
 
-## Features
-- Provide Sensor Data (Lidar, Cameras (depth, segmentation, rgb), GNSS, IMU)
-- Provide bboxes of vehicles.
-- Draw lidar points on RGB camera.
+Apart from that, we added some new features and updates:
+- Converted all the camera message types from ```sensor_msgs/Image``` to ```sensor_msgs/CompressedImage```.
+- Added support for ```ros map_server``` for ROS1 and ```nav2_map_server map_server``` for ROS2 to convert map images to ```nav_msgs/OccupancyGrid``` so that the map is able to be displayed on 3D panel in visualization tool like FoxGlove Studio or WebViz.
+- Added PCL maps on top of ```ROS1/ROS2 map_server``` to display more detailed maps on the 3D panel of your favorite visualization tool.
+- Added support to publish ego vehicle traversed path as ```nav_msgs/Path``` which itself is a list of waypoints ```geometry_msgs/PoseStamped```.
+- Added a separate ROS topic that publishes RGB camera images with 3D Bounding boxes and LiDAR points overlaid.
 
-## Getting started
 
-### Docker setup
+# How to use
+Autonomous vehicles require large-scale development and testing in a wide range of scenarios before they can be deployed. Moreover, huge amounts of data are required to train models for planning and perception, which can be costly and time consuming. To solve the problem, simulated datasets from the CARLA simulator can be a great solution but still could be time-consuming and might require diligence to validate the data gathering process.
+
+
+## CarlaFox
+We implemented [Carlafox](https://github.com/collabora/carlafox) - a web-based CARLA visualizer that can combine a number of ROS topics message streams, including support for text and custom shapes in one single visualization to solve the problem of monitoring the data gathering process or testing your perception/planning models.
+
+
+Carlafox eradicates the hassle of using the ros-bridge with CARLA and adds FoxGlove Visualization on top of that. Instead of following intricate steps like running CARLA server, starting the ros-bridge, and finally, starting FoxGlove or any other visualization tool. Carlafox pulls everything together and makes it a lot easier to get started.
+
+
+## ROS Bridge & CARLA
+Besides, you can also use the ros-bridge without Carlafox. 
 - Clone the ros-bridge repo with carla_msgs submodule.
 ```bash
  git clone --recurse-submodules https://github.com/makaveli10/ros-bridge.git
  cd ros-bridge/
 ```
 
-- Build docker image
+- Build docker image with ROS1/ROS2
 ```bash
- docker build -t carlafox .
+ docker build -t carlafox . --build-arg ROS_DISTRO=noetic
+                                        ROS_DISTRO=foxy

```

Comment:
```suggestion
 docker build -t carlafox . --build-arg ROS_DISTRO=noetic ROS_DISTRO=foxy
```

So that GitHub can render this in a single line and it's easier to copy/paste the command.